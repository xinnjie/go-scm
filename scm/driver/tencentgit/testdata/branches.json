[
    {
        "protected": true,
        "developers_can_push": true,
        "developers_can_merge": false,
        "name": "master",
        "commit": {
            "id": "652a91b83cf37786c64eb286f1efba453a15f34f",
            "message": "Add LICENSE",
            "parent_ids": [
                "dc238bca5c226bb7a2c7cc5bdbfc9930d6abd8b6"
            ],
            "authored_date": "2021-08-20T12:08:22+0000",
            "author_name": "xinnjie",
            "author_email": "wx_1600e990b7cb469a97087b334c94acee@git.code.tencent.com",
            "committed_date": "2021-08-20T12:08:22+0000",
            "committer_name": "xinnjie",
            "committer_email": "wx_1600e990b7cb469a97087b334c94acee@git.code.tencent.com",
            "title": "Add LICENSE",
            "scroll_object_id": null,
            "created_at": "2021-08-20T12:08:22+0000",
            "short_id": "652a91b8"
        },
        "description": "a repo for tencentgit client integration test",
        "branch_type": {
            "name": "Mainline",
            "color": "#CC0033"
        }
    }
]