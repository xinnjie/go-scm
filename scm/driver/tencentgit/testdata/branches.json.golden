[
    {
        "Name": "master",
        "Path": "refs/heads/master",
        "Sha": "652a91b83cf37786c64eb286f1efba453a15f34f"
    }
]